src/components/MessageList/MessageList.jsx

```diff
@@ -1,15 +1,13 @@
-import { useEffect, useRef } from 'react';
+import { useEffect } from 'react';
+import { isSameDay } from 'date-fns';
+
 import PropTypes from 'prop-types';
 import MessageItem from '@/components/MessageItem/MessageItem';
 import { MESSAGE_TYPES } from '@/constants/messageTypes.js';
 import { MessageListStyled } from './MessageListStyled.js';
+import DateSeparator from '../DateSeparator/DateSeparator.jsx';
```

Comment:
Lets rename a path to '@/components/DateSeparator/DateSeparator.jsx'

Reply:
done